### DISCUSSION TITLE
Question: any way to improve sort ordering of the class search?

### DISCUSSION THREAD

**Discussion by machty** [lsegal/yard#1554] [Ideas]
I have a Purchase class, and it'd be nice if searching "Purchase" would surface exact matches to the top... is there any way to configure this ordering? This seems like something that must be trolling other people... am I missing some obvious API to get around this?

<img width="347" alt="image" src="https://user-images.githubusercontent.com/81818/192604923-4e3eddaf-ba1c-4b4b-a656-62ef72b4c8c5.png">

**Comment by lsegal** [comment]
A pull request would be accepted to improve search ordering in the frontend.
